AppCenter/AppCenterTests/MSChannelGroupDefaultTests.m

```diff
@@ -39,18 +39,12 @@ - (void)setUp {
                                                                  batchSizeLimit:batchSizeLimit
                                                             pendingBatchesLimit:pendingBatchesLimit];
   self.sut = [[MSChannelGroupDefault alloc] initWithIngestion:self.ingestionMock];
+  self.sut.logsDispatchQueue = dispatch_get_main_queue();
 }
 
 - (void)tearDown {
   [super tearDown];
-
-  // Wait all tasks in tests.
-  XCTestExpectation *expectation = [self expectationWithDescription:@"tearDown"];
-  dispatch_async(self.sut.logsDispatchQueue, ^{
-    dispatch_suspend(self.sut.logsDispatchQueue);
```

Comment:
@clpolet 
Disagree with this change. It fixes the "chain actions" case
Restored with comments in #1491